src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -303,6 +318,12 @@ void SetParameter()
       Aux_Message( stdout, "  central density threshold for CCSN_CC_MaxRefine_LV2 = %13.7e\n", CCSN_CC_MaxRefine_Dens2  ); }
       Aux_Message( stdout, "  central density factor for reducing dt              = %13.7e\n", CCSN_CC_CentralDensFac   );
       Aux_Message( stdout, "  reduced dt near bounce                              = %13.7e\n", CCSN_CC_Red_DT           );   }
+      Aux_Message( stdout, "  mode for rotational profile                         = %d\n",     CCSN_CC_Rot              );
+      if ( CCSN_CC_Rot == 1 ) {
+      Aux_Message( stdout, "  characteristic rotational radius R_0                = %13.7e\n", CCSN_CC_Rot_R0           );
+      Aux_Message( stdout, "  central rotational velocity Omega_0                 = %13.7e\n", CCSN_CC_Rot_Omega0       ); }
```

Comment:
- [ ] Replace "rotational velocity" with "angular frequency"
- [ ] Specify the units

Reply:
I updated the output message.